Dikke Ploaten/Services/Database.swift

```diff
@@ -11,10 +11,13 @@ import ObjectMapper
 
 class Database {
 	
+	//Firebase
 	let db = Firestore.firestore()
 	
+	static let shared = Database()
```

Comment:
Je hebt hier de initializer niet private gemaakt, dat is een onderdeel van het singleton pattern.

Reply:
Hoe roep je deze dan op in je controllers?

Reply:
Is dit gwn private init {} toevoegen?

Reply:
Correct.